saleor/graphql/attribute/utils.py

```diff
@@ -513,8 +515,8 @@ def save(cls, instance: T_INSTANCE, cleaned_input: T_INPUT_MAP):
 
 
 def get_variant_selection_attributes(qs: "QuerySet"):
-    return qs.filter(
-        input_type__in=AttributeInputType.ALLOWED_IN_VARIANT_SELECTION,
+    return qs.select_related("attributevariant").filter(
```

Comment:
From what I see we currently have two functions `get_variant_selection_attributes` - one does filtering in DB and the other filters in Python. Do you think we could refactor them and have a single function instead?

Reply:
As we spoke, these functions has common name only by some sad accident, they are operate on different types of data (`QuerySet` vs some Eager collection of tuples). Anyway, I have add typing, so tools like `mypy` or even `PyRight lsp` will cover that.